src/sugar3/activity/activityfactory.py

```diff
@@ -345,4 +381,4 @@ def error_handler_cb(error):
         # main window, see http://bugs.sugarlabs.org/ticket/1447#comment:19
         shell.NotifyLaunchFailure(activity_id,
                                   reply_handler=reply_handler_cb,
-                                  error_handler=error_handler_cb)
+                                  error_handler=error_handler_cb)
```

Comment:
Please remove the No new line at EOF, you can do so with `echo >> activityfactory.py`.

Reply:
Also see if you can figure out how it happened; we don't want unnecessary changes, it consumes reviewing time, and makes `git bisect` and `git blame` slightly more interesting than we can emotionally deal with.